packages/xrpl/src/models/transactions/AMMBid.ts

```diff
@@ -0,0 +1,85 @@
+import { ValidationError } from '../../errors'
+import { Amount } from '../common'
+
+import { BaseTransaction, isAmount, validateBaseTransaction } from './common'
+
+const MAX_AUTH_ACCOUNTS = 4
+
+interface AuthAccount {
+  AuthAccount: {
+    Account: string
+  }
+}
+
+/**
+ * AMMBid is used for submitting a vote for the trading fee of an AMM Instance.
+ *
+ * Any XRPL account that holds LPToken for an AMM instance may submit this
+ * transaction to vote for the trading fee for that instance.
+ */
+export interface AMMBid extends BaseTransaction {
+  TransactionType: 'AMMBid'
+
+  /**
+   * A hash that uniquely identifies the AMM instance. This field is required.
+   */
+  AMMID: string
+
+  /**
+   * This field represents the minimum price that the bidder wants to pay for the slot.
+   * It is specified in units of LPToken. If specified let MinSlotPrice be X and let
+   * the slot-price computed by price scheduling algorithm be Y, then bidder always pays
+   * the max(X, Y).
```

Comment:
Reworded to simplify 
```suggestion
   * This field represents the minimum price that the bidder wants to pay for the slot.
   * It is specified in units of LPToken. If specified, the bidder always pays the max between the 
   * MinSlotPrice and the slot-price computed by the price scheduling algorithm.
```